mon-pix/app/services/modulix-auto-scroll.js

```diff
@@ -44,4 +48,8 @@ export default class ModulixAutoScroll extends Service {
   #getWindowScrollY() {
     return window.scrollY;
   }
+
+  #getNavbar() {
+    return document.querySelector('#module-navbar');
```

Comment:
Suggestion : aussi simple (et certainement plus performant) avec
```suggestion
    return document.getElementById('module-navbar');
```